src/engine/source/api/include/api/test/sessionManager.hpp

```diff
@@ -0,0 +1,91 @@
+#ifndef _SESSIONS_MANAGER_HPP
+#define _SESSIONS_MANAGER_HPP
+
+#include <ctime>
+#include <iostream>
+#include <mutex>
+#include <optional>
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+#include <fmt/format.h>
+
+#include <error.hpp>
+
+namespace api::sessionManager
+{
+
+constexpr auto FILTER_CONTENT_FORMAT =
+    R"({{"name": "{}", "check":[{{"~TestSessionName":"{}"}}]}})"; ///< Filter content format, where '{}' is the session
```

Comment:
improve - desing: isn't valid json. Use https://jsonformatter.curiousconcept.com/# for verification

Reply:
It is formatted like that so it is processed by `fmt::format` afterward